src/core/core.scale.js

```diff
@@ -1255,16 +1259,20 @@ class Scale extends Element {
 			ctx.fillStyle = tickFont.color;
 			ctx.textBaseline = 'middle';
 			ctx.textAlign = item.textAlign;
+			ctx.strokeStyle = tickFont.strokeStyle;
+			ctx.lineWidth = tickFont.lineWidth;
 
 			label = item.label;
 			y = item.textOffset;
 			if (isArray(label)) {
 				for (j = 0, jlen = label.length; j < jlen; ++j) {
 					// We just make sure the multiline element is a string here..
+					ctx.strokeText('' + label[j], 0, y);
 					ctx.fillText('' + label[j], 0, y);
 					y += tickFont.lineHeight;
 				}
 			} else {
+				ctx.strokeText(label, 0, y);
```

Comment:
I'm assuming the performance impact of this is negligible. Is this correct?

Reply:
Well, in my opinion, the impact it's negligible. But I did a quick test with 1,000 texts with and without stroke: https://codepen.io/joseraul/pen/dyyEvdZ, Some examples:
```
1,000 strokes: 10 miliseconds
1,000 no strokes: 4 miliseconds
```
```
1,000 strokes: 9 miliseconds
1,000 no strokes: 7 miliseconds
```
So we can decide with data if it worth or not to add the conditional `tickFont.lineWidth > 0`

Reply:
Thank you for testing. That seems fine to me. Normally a chart will have only a small number of labels